test/Ecosystem.jl

```diff
@@ -0,0 +1,139 @@
+using Test
+using ADRIA
+using ADRIA.Distributions
+
+@testset "Truncated normal mean" begin
+
+    means::Vector{Float64} = Vector{Float64}([0.0, 0.5, 1.5, 2.5, 4.0, 5.0])
+    stdev::Vector{Float64} = Vector{Float64}([0.1, 0.25, 0.5, 1.0, 2.0, 5.0, 10.0])
+    lower_bounds::Vector{Float64} = Vector{Float64}([0.0, 0.5, 1.5, 3.0, 7.0, 10.0])
+    bound_widths::Vector{Float64} = Vector{Float64}([0.25, 0.5, 1.0, 2.0, 5.0, 10.0])
```

Comment:
Is there reasoning behind these specific values?

Otherwise, I'd prefer we check a range of randomly generated combinations instead of (or in addition to) these.